src/convertToObject.js

```diff
@@ -3,16 +3,32 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const resultObj = {};
+  const stylesArr = sourceString.split(';');
+
+  for (const property of stylesArr) {
+    const splittesProperty = property.split(':');
+
+    const [key, value] = splittesProperty;
+
+    if (key.trim().length
+    && value.trim().length > 0) {
+      resultObj[key.trim()] = value.trim();
+    }
```

Comment:
let's align your code 
```suggestion
    const splittesProperty = property.split(':');
    const [key, value] = splittesProperty;

    if (
      key.trim().length &&
      value.trim().length
    ) {
      resultObj[key.trim()] = value.trim();
    }
```

Reply:
okay